packages/react-components/react-list-preview/src/components/List/List.types.ts

```diff
@@ -1,17 +1,37 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import * as React from 'react';
+import type { ComponentProps, ComponentState, Slot, SelectionMode, SelectionItemId } from '@fluentui/react-utilities';
+import { ListSelectionState } from '../../hooks/types';
+
+export type ListLayout = 'horizontal' | 'vertical' | 'grid';
```

Comment:
As I see it's used once inside:

<img width="377" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/eab4f04c-4dcc-4069-93c7-a64dd5a7e42a">

Can we avoid exporting & exposing this type?

Reply:
yea it used to be an enum, thats why it was exported. That doesn't make sense anymore.